demos/smart_classroom_demo/cpp_gapi/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2018-2019 Intel Corporation
```

Comment:
Extend tests with the new demo https://github.com/openvinotoolkit/open_model_zoo/blob/9ca5dbeff80464bf5728e8be25daedfe9a9208d7/demos/tests/cases.py#L353

Reply:
Added